source/Operation Guides/InterFlux Masternodes/Using Masternode Setup Tool/masternode-setup-tool.rst

```diff
@@ -0,0 +1,309 @@
+************
+Introduction
+************
+
+This guide details the process required for joining the Cirrus
+Sidechain as a Block Producing Node, known as an InterFlux Masternode using the Masternode Setup Tool. The request to join the Cirrus Sidechain as a block
+producer, requires you to setup the Masternode Setup tool, therefore, a request to join the Cirrus Sidechain will be
+made once the Masternode is fully configured.
+
+Pre-Requisites
+==============
+
+To join as a InterFlux Masternode, there are a number of criteria
+that must be met.
+
+Wallet Requirements
+-------------------
+
+You must know the mnemonic and passphrase (if applicable) for a STRAX
+Wallet that contains the required collateral and fee.
+
+-  **Collateral Amount:** 100,000 STRAX
+
+-  **Fee Amount:** 501 STRAX
+
+
+Sidechain Masternode Hardware Requirements
+------------------------------------------
+
+The below hardware requirements are a minimum.
+
+-  2x vCPU / Dual Core
+
+-  8GB RAM
+
+-  50GB SSD / HDD
+
+Sidechain Masternode Software Requirements
+------------------------------------------
+
+There are only three pre-requisites software installations that need to
+take place before operating a InterFlux Masternode.
+
+.NET Core SDK
```

Comment:
I think we don't need .NET SDK to run the node, but only .NET Runtime

Reply:
I think that the tool prompts for download during the installation process, so should cater for that.